MenuApplets/ResourceGraph/main.cpp

```diff
@@ -118,6 +118,17 @@ class GraphWidget final : public GUI::Frame {
         }
     }
 
+    virtual void enter_event(Core::Event& event) override
+    {
+        // FIXME: Remember if tooltip is open and keep updating it while it's open.
```

Comment:
How about we just `set_tooltip()` unconditionally after every update?

Reply:
I think that's better. But the main problem is that even static tooltips don't show up for MenuApplets :)

Reply:
Oh I see! I guess because they look at `Window::rect()` which is empty. There is `Window::rect_in_menubar()` for applets. I'm not sure why those are separate APIs really. Maybe it should always be `Window::rect()` from the LibGUI side, makes things easier.

Reply:
Yep, see FIXME in https://github.com/SerenityOS/serenity/pull/3120/commits/f83f324b761f2a4b51aa523fe31e859c4d10ee6b :) @linusg said he's interested in looking at this on IRC :)

Reply:
Oh duh, it says so right there! :)